__tests__/app.test.js

```diff
@@ -38,4 +40,35 @@ describe('GET /api/topics', () => {
       });
       });
     });
-  });
\ No newline at end of file
+  });
+
+  //STILL NEED TO DO TASK 4!
+
+  describe('GET /api/articles', () => {
+    test('200: serves an array of all articles', () => {
+      return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({body}) => {
+        const { articles } = body;
+        expect(Array.isArray(articles)).toBe(true);
```

Comment:
This assertion is redundant as you use an array method and (correctly) check the length - both of these things could not be done and the test would fail if `articles` was not an array.